components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -755,6 +767,30 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
                 workspaceId={this.props.workspaceId}
             >
                 {statusMessage}
+                {this.state.redirected && (
+                    <>
+                        <Alert type="info" className="mt-4 w-112">
```

Comment:
Do we use Alerts in other places as well? Or should we rather go with some form of "warning modal"?
Asking because I feel alerts are a bit too invasive...

Reply:
I'm stupid, ignore my previous comment please :see_no_evil: